java-client/src/main/java/org/opensearch/client/opensearch/_types/aggregations/MultiTermsAggregation.java

```diff
@@ -37,25 +37,31 @@
 import org.opensearch.client.json.JsonpMapper;
 import org.opensearch.client.json.ObjectBuilderDeserializer;
 import org.opensearch.client.json.ObjectDeserializer;
+import org.opensearch.client.opensearch._types.aggregations.BucketSortAggregation.Builder;
```

Comment:
Does not look like the right import

Reply:
Thanks for the finding. I removed this line.